website/docs/user_guide/rbac.md

```diff
@@ -123,3 +123,15 @@ A user group consists of the following:
 Groups do nothing on their own. They must be given a role on a project to assign permissions. You can assign both standard roles and custom project roles to groups.
 
 While a user can only have one role in a given project, a user may belong to multiple groups, and each of those groups may be given a role on a project. In the case where a given user is given permissions to a project through more than one group, the user will inherit most permissive permissions of all their groups in that project.
+
+## User Group SSO Integration
```

Comment:
This is a nice paragraph, but I'm lacking information on what the options are for syncing. The how-to guides reference the "Group Field JSON path" thing. What does that mean? Are there other things that need to be configured? This is the place to introduce and explain them!  😄

Reply:
I've added a bunch of text, will you check it out?

Reply:
Sure will!